.github/pull_request_template.md

```diff
@@ -1,57 +1,17 @@
-# Overall Description
+# Description
 
-Please write out a short description of what this pull request does. Also, please go through the following sections of checklists and include/omit any checklists as needed and add additional detail there.
+Describe your changes and why you're making them. Please include the context, motivation, and relevant dependencies.
 
-## Checklist for all PRs
+Resolves # [issue]
 
-- [ ] Run `pre-commit run --all-files` to make sure markdown/lint passes
-- [ ] Link this pull request to all issues that it will close using keywords (see GitHub docs about [Linking a pull request to an issue using a keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)). Also mention any issues that are partially addressed or are related.
+## Type of change
 
-## agencies.yml changes checklist
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation
+- [ ] agencies.yml
```

Comment:
Since GitHub will show a progress bar based on how many of these checklist items are checked, I don't think this checklist feature is appropriate here since most PRs will not check off each of these items.